src/window.vala

```diff
@@ -53,7 +53,7 @@ public class WriteAs.MainWindow : Gtk.ApplicationWindow {
             // TODO This word count algorithm may be quite naive
             //      and could do improvement.
             var word_count = canvas.buffer.text.split(" ").length;
-            header.subtitle = ngettext("%i word","%i words",word_count).printf(word_count);
+            header.subtitle = ngettext(_("%i word"),_("%i words"),word_count).printf(word_count);
```

Comment:
```suggestion
            header.subtitle = ngettext("%i word","%i words",word_count).printf(word_count);
```

Actually this line is already translatable if we don't add `_()`.

---

In English, there're 2 forms of a noun depends on numbers:

**In the case of 1**:
e.g. "1 **file**"

**More than 1**:
e.g. "2 **files**"

However, some language has more than 2 plural forms. Also, how plural forms built may differ in each language. So, if a translation string contains variables (like `%s`), we often use `ngettext` instead of just adding `_()`.

There's a official documentation which is more detailed and may be easier to understand at https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html

Reply:
Sure! I didn't know that :smile: